pennylane_lightning/core/src/simulators/lightning_kokkos/utils/UtilKokkos.hpp

```diff
@@ -32,6 +32,26 @@ namespace Pennylane::LightningKokkos::Util {
 
 constexpr std::size_t one{1};
 
+template <typename T>
```

Comment:
Introduced convenient utility function to go back and forth between C++ and Kokkos.